src/server/middleware/liveReloadMiddleware.js

```diff
@@ -6,21 +6,19 @@
 /**
  * Live reload middleware
  */
-function liveReloadMiddleware(compiler) {
-  let watchers = [];
```

Comment:
Can we revert this change? I would like watchers to be declared within the middleware itself. I see that primary reason for adding it was to share it between this middleware and `reloadAppMiddleware` (singleton). In this case, I believe you can just add respective if statement (`if (req.path === '/reloadapp')`) below the one declared on line 17. It's okay for one middleware to operate on more than one path (see `devToolsMiddleware.js`)